tests/ffi/tcp_max_syn_backlog/test.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+# shellcheck disable=SC1091
+. ../common/prepare.sh
+
+# Declare global variables and initialize to 0
```

Comment:
Hi, nice patch.

Small comment. Probably these global variables need to be moved to main.fmf as env vars. :)

```
summary: Check that changes to network settings inside QM doesn't change network settings
    on Host.
test: /bin/bash ./test.sh
environment:
QM_NET_IPV4_TCP_MAX_SYN_BACKLOG_VALUE=0
HOST_DEFAULT_NET_IPV4_TCP_MAX_SYN_BACKLOG_VALUE=0
HOST_NET_IPV4_TCP_MAX_SYN_BACKLOG_VALUE=0
duration: 10m
tag: ffi
framework: shell
id: 7b86ccc3-072d-40b6-8463-3f24ceaba9b2
```

than remove it from the bash script and should be automatically available in the bash script, please test it to make sure it works.

@Yarboa could you please review if you have time before flying?

Reply:
Fixed and changed README.md.

Reply:
Moving global variables to main.fmf causing for test failure on testing-farm:
Failed to discover tests, TMT metadata are absent or corrupted. Search for '[E] [test-schedule-tmt' on the artifacts page for details.
If possible, I'd prefer to leave them inside the test.sh.

I've tried with this syntax in main.fmf and commented out these variables in test.sh:
summary: Check that changes to network settings inside QM doesn't change network settings
    on Host.
test: /bin/bash ./test.sh
environment:
QM_NET_IPV4_TCP_MAX_SYN_BACKLOG_VALUE=0
HOST_DEFAULT_NET_IPV4_TCP_MAX_SYN_BACKLOG_VALUE=0
HOST_NET_IPV4_TCP_MAX_SYN_BACKLOG_VALUE=0
duration: 20m
tag: ffi
framework: shell
id: 7b86ccc3-072d-40b6-8463-3f24ceaba9b2